### ISSUE TITLE
Stuck on "Dashboard" (AF 5.2.2. DNN 7.2.1)

### ISSUE THREAD

**Issue by rocketmannate** [ActiveForums/ActiveForums#107]
I have just installed Active Forums 5.2.2 onto a site that is running on DNN 7.2.1. When I enter the control panel, I land on the "dashboard" tab. When I try to go to any other tab (Forums, Templates, Filters, etc...), nothing happens. Also, when I try to create a new forum or a new forum group, nothing happens.

I tried uninstalling and reinstalling, but to no avail. 

Any ideas? Or is this a new issue?

**Comment by artfuld** [comment]
@rocketmannate do you still have this problem?

**Comment by CRATAAdmin** [comment]
I too am having this same issue. We are running DNN 07.04.02 and AF 6.3.0. We've tried uninstalling and reinstalling a fresh version of the mod, and I just adjusted the edit skin to Gravity, but to no avail.  When I go to the control panel and select the Create New Group or Forum links, the dashboard main menu ribbon simply reloads. The same occurs when view a particular sample forum and attempting to edit it. I am unable to create or edit any forums.

Was this resolved?

**Comment by artfuld** [comment]
@CRATAAdmin do you get any errors in the event viewer?

**Comment by CRATAAdmin** [comment]
No, unfortunately, I'm not.

**Comment by artfuld** [comment]
Do you get any javascript-errors? Is it possible for me to see this happening?

**Comment by CRATAAdmin** [comment]
My first thought was that it was a JS issue, but no, I'm not getting any errors. Can you provide me with your email address so that I can send you a login?

**Comment by artfuld** [comment]
jocke@laserdome.tk

**Comment by CRATAAdmin** [comment]
Thank you! I've sent you an email =)

**Comment by CRATAAdmin** [comment]
Your help led us to the issue, thank you! We had pipes blocked on the site, so when AF tried to run /Default.aspx?cpview=manageforums_forumeditor&params=0|G, we weren't allowing it on the server. We corrected this on our end and I can now create and edit forums.

I really appreciate the help!

**Comment by artfuld** [comment]
Glad I could help!

2015-11-20 16:05 GMT+01:00 CRATAAdmin <notifications@github.com>:
